files/qsub.pl

```diff
@@ -349,11 +347,16 @@ sub make_command
     push(@command, "--ntasks=$node_opts->{task_cnt}") if $node_opts->{task_cnt};
     push(@command, "--nodelist=$node_opts->{hostlist}") if $node_opts->{hostlist};
 
-    push(@command, "--chdir=$workdir") if $workdir;
-
     push(@command, "--mincpus=$res_opts->{ncpus}") if $res_opts->{ncpus};
     push(@command, "--ntasks-per-node=$res_opts->{mppnppn}")  if $res_opts->{mppnppn};
 
+    if ($workdir) {
```

Comment:
why is this needed? what was not working with the old code?

Reply:
Without this, `--chdir=$HOME` is always used, which is wrong if an alternate location is passed with `qsub -d <path>`